src/helpers/discord_utils.ts

```diff
@@ -108,33 +108,27 @@ export async function sendOptionsMessage(message: Eris.Message<Eris.GuildTextabl
     const goalMode = guildPreference.isGoalSet();
     const guessTimeoutMode = guildPreference.isGuessTimeoutSet();
 
-    let cutoffString = `between the years ${guildPreference.getBeginningCutoffYear()} - ${guildPreference.getEndCutoffYear()}`;
-    let genderString = `${guildPreference.getSQLGender()} artists`;
-    let groupsString = groupsMode ? `${guildPreference.getGroupNames().join(", ")}` : null;
-    let limitString = `${Math.min(totalSongs, guildPreference.getLimit())}`;
-    let seekTypeString = `${guildPreference.getSeekType()}`;
-    let modeTypeString = `${guildPreference.getModeType() === MODE_TYPE.BOTH ? "song or artist" : guildPreference.getModeType()}`;
-    let goalString = `${guildPreference.getGoal()}`;
-    let guessTimeoutString = `${guildPreference.getGuessTimeout()}`;
-
-    cutoffString = updatedOption == GameOption.CUTOFF ? bold(cutoffString) : codeLine(cutoffString);
-    genderString = updatedOption == GameOption.GENDER ? bold(genderString) : codeLine(genderString);
-    limitString = updatedOption == GameOption.LIMIT ? bold(limitString) : codeLine(limitString);
-    if (groupsString && groupsString.length > 400) {
-        groupsString = `${groupsString.substr(0, 400)} and many others...`;
```

Comment:
Don't we still need to limit how many groups get printed because of the length limit of an embed?

Reply:
Fixed